.gitignore

```diff
@@ -0,0 +1 @@
+docs/_intermediate
```

Comment:
Mandatory newline warning :D

Reply:
Thanks! I always manage to miss these 🤦‍♂️